src/components/shared/Dashboard/DashboardNewDeploy.tsx

```diff
@@ -0,0 +1,80 @@
+import { BigNumber } from '@ethersproject/bignumber'
+import { Trans } from '@lingui/macro'
+import { Button } from 'antd'
+import FeedbackPromptModal from 'components/v1/V1Project/modals/FeedbackPromptModal'
+
+import { useState } from 'react'
+
+import { useHistory } from 'react-router-dom'
+
+import { layouts } from 'constants/styles/layouts'
+import { padding } from 'constants/styles/padding'
+
+export default function DashboardNewDeploy({
+  handle,
+  projectId,
+  owner,
+  initialFeedbackModalOpen = false,
+  isV2 = false,
+}: {
+  handle?: string
+  projectId?: BigNumber
+  owner: string | undefined
+  initialFeedbackModalOpen?: boolean
+  isV2?: boolean
```

Comment:
These last 2 props will be unnecessary once the feedback modal is removed

Reply:
Have reworked - now takes in optional `name` and `projectId`. Preference is to use `name` and fallback to `projectId` if a `name` cannot be found.

`name` is used in v1 for `handle` and in v2, will use the metadata name if available, or fallback to projectId